sentry/src/main/java/io/sentry/SentryEvent.java

```diff
@@ -14,7 +14,10 @@
 public final class SentryEvent implements IUnknownPropertiesConsumer {
   private SentryId eventId;
   private final Date timestamp;
-  private transient Throwable throwable;
+
+  /** The captured Throwable */
```

Comment:
`m`: I think this change could be in another PR. It also nicer for the changelog IMO, but the change itself looks good. 👍

Reply:
but this is the title of the PR: `enha: make getThrowable public and improve set contexts`
I know its a mix of 2 changes, but both changes unblock this https://github.com/getsentry/sentry-docs/pull/2460 so it made sense to be done together, happy to split them if you think thats important.

Reply:
I don't have a strong opinion on this, just thought it would be cleaner.

Reply:
ok, i will keep as it is now because they are related to the docs, but next time i will split them up, thanks